XPlat/DropboxApiV2/nuget/Xamarin.Dropbox.Api.Android.nuspec

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Xamarin.Dropbox.Api.Android</id>
+    <title>Xamarin.Dropbox.Api.Android</title>
+    <version>4.5.3</version>
+    <authors>Xamarin</authors>
+    <owners>Xamarin</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Xamarin.Android Helper classes for Xamarin.Dropbox.Api.Core</description>
+    <copyright>Copyright © Xamarin. All Rights Reserved.</copyright>
+    <licenseUrl>https://components.xamarin.com/license/dropboxapiv2</licenseUrl>
```

Comment:
Point these 2 urls to Github where the file will be on master.  Do this for all 2 nuspec files.

Reply:
Updated to point to Github main repo and licence files in all of the nuspec files